gostatsd.go

```diff
@@ -67,8 +67,8 @@ func main() {
 			return
 		}
 		defer func() {
-			if err := f.Close(); err != nil {
-				augmentErr(&exitErr, fmt.Errorf("Failed to close profile file: %v", err))
+			if closeErr := f.Close(); closeErr != nil {
```

Comment:
I really think this is too much. I don't think we care about the error here. If it is because of the linter, then we can disable warnings for the `Close` case. This is too verbose imho.